zproject/prod_settings_template.py

```diff
@@ -692,3 +692,7 @@
 
 ## Similarly if you want to set a Privacy Policy.
 # PRIVACY_POLICY = '/etc/zulip/privacy.md'
+
+## We use this API provided by Kroki to render
+## diagrams from textual descriptions.
+KROKI_URL = "https://kroki.io/"
```

Comment:
Let's leave adding this to `prod_settings_template.py` for the end; for this commit:
* You can set that URL in `dev_settings.py` and set `None` in `default_settings.py`, so that the feature is disabled except in the development environment, so that we can incrementally merge infrastructure commits without changing anything in production.
* We should call it `KROKI_SERVER_URL`, in analogy to `JITSI_SERVER_URL`.